tasks/species_typing/task_tbprofiler_output_parsing.wdl

```diff
@@ -16,65 +18,145 @@ task tbprofiler_output_parsing {
+                  "M_DST_K01_LFX": {"gyrA": "M_DST_K02_gyrA", 
+                                    "gyrB": "M_DST_K03_gyrB"},
+                  "M_DST_L01_BDQ": {"Rv0678": "M_DST_L02_Rv0678", 
+                                    "atpE": "M_DST_L03_atpE",
+                                    "pepQ": "M_DST_L04_pepQ", 
+                                    "mmpL5": "M_DST_L05_mmpL5",
+                                    "mmpS5": "M_DST_L06_mmpS5"},
+                  "M_DST_M01_CFZ": {"Rv0678":"M_DST_M02_Rv0678", 
+                                    "pepQ": "M_DST_M03_pepQ",
+                                    "mmpL5":"M_DST_M04_mmpL5", 
+                                    "mmpS5": "M_DST_M05_mmpS5"},
+                  "M_DST_N01_LZD": {"rrl": "M_DST_N02_rrl", 
+                                    "rplC": "M_DST_N03_rplC"}
                 }
 
-    # lookup dictionary - gene to resistance (https://github.com/jodyphelan/tbdb/blob/master/tbdb.csv)
-    gene_to_resistance = {"ahpC":["isoniazid"], "ald":["cycloserine"], "alr": ["cycloserine"],
-                          "ddn": ["delamanid"], "eis": ["amikacin", "kanamycin"], "embA": ["ethambutol"],
-                          "embB": ["ethambutol"], "embC": ["ethambutol"], "embR": ["ethambutol"],
-                          "ethA": ["ethionamide"], "ethR": ["ethionamide"], "fabG1": ["ethionamide", "isoniazid"],
-                          "fbiA": ["delamanid"], "fgd1": ["delamanid"], "folC": ["para-aminosalicylic_acid"],
-                          "gid": ["streptomycin"], "gyrA": ["ciprofloxacin", "fluoroquinolones", "levofloxacin",
-                          "moxifloxacin", "ofloxacin"], "gyrB": ["moxifloxacin","ciprofloxacin","fluoroquinolones",
-                          "levofloxacin", "ofloxacin"], "inhA": ["ethionamide", "isoniazid"],
-                          "kasA": ["isoniazid"], "katG": ["isoniazid"], "panD": ["pyrazinamide"],
-                          "pncA": ["pyrazinamide"], "ribD": ["para-aminosalicylic_acid"], "rplC": ["linezolid"],
-                          "rpoB": ["rifampicin"], "rpoC": ["rifampicin"], "rpsA": ["pyrazinamide"],
-                          "rpsL": ["streptomycin"], "rrl": ["linezolid"], "rrs": ["streptomycin", "amikacin",
-                          "aminoglycosides", "capreomycin", "kanamycin"], "Rv0678": ["bedaquiline",
-                          "clofazimine"], "thyA": ["para-aminosalicylic_acid"], "thyX": ["para-aminosalicylic_acid"],
-                          "tlyA": ["capreomycin"]
+    # lookup list - the genes to be considered for the LIMS report
+    GENES_FOR_LIMS = ["katG", "fabG1", "inhA", "ethA", "rpoB", "pncA", "embA", "embB", "rrs", "eis", 
+                    "tlyA", "gyrA", "gyrB", "Rv0678", "atpE", "pepQ", "mmpL5", "mmpS5", "rrl", "rplC"
+                    ]
+
+    # lookup dictionary - gene to antimicrobial drug name (https://github.com/jodyphelan/tbdb/blob/master/tbdb.csv)
+    GENE_TO_ANTIMICROBIAL_DRUG_NAME = {"ahpC":["isoniazid"], 
```

Comment:
Missing https://github.com/jodyphelan/tbdb/blob/master/tbdb.watchlist.csv

Reply:
implemented in [e00fe38](https://github.com/theiagen/public_health_bioinformatics/pull/97/commits/e00fe3804b2608a80f090d051964f5093fa309e3)